src/clean.c

```diff
@@ -189,6 +189,27 @@ static bool is_all_digits(const char *s)
 	return true;
 }
 
+static bool is_all_xdigits(const char *s)
+{
+	for (; *s; s++) {
+		if (!isxdigit(*s)) {
+			return false;
+		}
+	}
+	return true;
+}
+
+static bool is_hashed_manifest(const char UNUSED_PARAM *dir, const struct dirent *entry)
+{
+	static const char prefix[] = "Manifest.";
+	static const size_t prefix_len = sizeof(prefix) - 1;
+	if (strncmp(entry->d_name, prefix, prefix_len)) {
+		return false;
+	}
```

Comment:
Can you call the is_manifest function instead of using the above code?

Reply:
nice. yes.